src/gui_common/CreditsScroll.cs

```diff
@@ -646,7 +647,7 @@ private int CreateTeamNameList(int offset, string team, IEnumerable<string> peop
         // If past team leads need to be marked might need to use rich text label or some other approach there.
         // And this method might need to be split into separate implementations as not all places would need that.
         var memberLabel = CreateDynamicPart(offset, people, columns);
-        offset += (int)memberLabel.Height + ExtraOffsetAfterTeam;
+        offset += (int)Math.Round((memberLabel.Height + ExtraOffsetAfterTeam) / columns);
```

Comment:
Can you explain *how* this works? The number of columns should have *no* impact on height value. The height value is the height of the Godot control node here, the control node shouldn't multiply its height by the number of columns.